src/js/components/search/topFilterBar/filterGroups/AgencyFilterGroup.jsx

```diff
@@ -43,6 +43,13 @@ export default class AgencyFilterGroup extends React.Component {
 
         agencies.forEach((value) => {
             let agencyTitle = value.subtier_agency.name;
+
+            if (value.agencyType === 'subtier') {
```

Comment:
This is still happening.
![screen shot 2017-08-14 at 3 10 40 pm](https://user-images.githubusercontent.com/3594363/29287230-c3ecaf2e-8102-11e7-8f40-ca55fbe7cee0.png)